assets/clickhouse_native_json/main.py

```
@@ -0,0 +1,162 @@
+            id int, \
+            ctx String, \
+            msg String, \
+            attr JSON \
+            """
+        else:
+            table_fields = """ \
+            json JSON \
+            """
+
+        params = [
+                "ENGINE = MergeTree"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)